samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -292,7 +277,7 @@ static void provision_credentials(void)
 
 	security_instance = find_server_security_instance();
 	if (security_instance == -1) {
-		LOG_ERR("No security instance found");
+		LOG_ERR("Error %d: no security instance found", -ENODEV);
```

Comment:
A tautology, the information that this is an error has been included on the macro itself. I would keep these kind of changes to minimum to make it easier to concentrate on the changes which are the reason this PR got created.

Reply:
I think most of the tautologies are fixed now